questionnaire-en.json

```diff
@@ -168,7 +159,7 @@
         },
         {
           "correctAnswer": "Yes",
-          "question": "Is the Open Source Liaison function publicly identified (e.g. via an email address and/or the Linux Foundation\u0027s Open Compliance Directory)?",
+          "question": "Is the external Open Source compliance contact publicly identified (e.g. via an email address or the Linux Foundation Open Compliance Directory)?",

```

Comment:
Did we change the definitions for Liaison function?  I recall this being discussed and wanted to make sure it was OK with the participants in the prior discussion.  @MarkGisi do you recall the discussion?

Reply:
There is no Open Source Liaison function in the Definitions of OpenChain Specification 2.0, so this terminology should not be included in the questionnaire. It does not read to the spec and it assumes knowledge that the audience cannot be expected to have.
https://wiki.linuxfoundation.org/_media/openchain/openchainspec-2.0.pdf